README.md

```diff
@@ -1,3 +1,37 @@
 # Unity Rust Native
 
 A system for compiling native rust plugins and hot reloading in the Unity Engine.
+
+## Installation
+
+The RustNative plugin needs to be installed in your Unity project for it to work.
+
+If you are starting a new project, create one or open an existing project you wish to work with.
+
+As demonstrated below, in the `Project` pane, right click on `Assets` and navigate to the `Import Package` submenu and choose `Custom Package...`.
+
+![](https://github.com/nerdo/UnityRustNative/assets/1031502/45a5437b-db1a-4785-b9dc-05215f22c77b)
```

Comment:
Looks like you linked the images to an external repo. To make sure everything stays together, it would help to also push these files in the PR and make them relative paths